packages/block-editor/src/date-time-picker/attributes.js

```diff
@@ -0,0 +1,32 @@
+export default {
+	clientId: {
+		type: 'string',
+		default: null,
+	},
+	label: {
+		type: 'string',
+		default: '',
+	},
+	date: {
+		type: 'string',
+		default: null,
+	},
+	time: {
+		type: 'string',
+		default: null,
+	},
```

Comment:
I guess you don't need those two attributes, right?
On the Renderer version, the selected values can be stored in the component's local state and sent to the backend when the page is submitted.